packages/templates/src/raw/self-care-guide/index.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { __, _x } from '@web-stories-wp/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { default as template } from './template';
+
+export default {
+  title: _x('Self-care Guide', 'template name', 'web-stories'),
+  tags: [
+    _x('Health & Wellness', 'template keyword', 'web-stories'),
+    _x('Health', 'template keyword', 'web-stories'),
+    _x('Self Care', 'template keyword', 'web-stories'),
+    _x('Informative', 'template keyword', 'web-stories'),
+    _x('White', 'template keyword', 'web-stories'),
+  ],
+  colors: [
+    { label: _x('Studio White', 'color', 'web-stories'), color: '#F5F4EE' },
+    {
+      label: _x('Cavernous Gray', 'color', 'web-stories'),
+      color: '#535353',
+    },
+    { label: _x('Gold Ochre', 'color', 'web-stories'), color: '#b17417' },
+    { label: _x('Sea Turtle Brown', 'color', 'web-stories'), color: '#5e564a' },
+    { label: _x('Cove Blue', 'color', 'web-stories'), color: '#7a87a8' },
+    { label: _x('River Bed Blue', 'color', 'web-stories'), color: '#464c5a' },
+    { label: _x('Velvet Green', 'color', 'web-stories'), color: '#588061' },
+    {
+      label: _x('Forest Fern Green', 'color', 'web-stories'),
+      color: '#424e44',
+    },
+    {
+      label: _x('Olive Garden Green', 'color', 'web-stories'),
+      color: '#878c52',
+    },
+    { label: _x('Old Bamboo Green', 'color', 'web-stories'), color: '#63654f' },
+    { label: _x('White', 'color', 'web-stories'), color: '#ffffff' },
+  ],
```

Comment:
THat's quite a lot of colors. Can we reduce that list a little bit? Some colors are very similar.

Reply:
For colours, these were provided in Figma, do we want to run it by design team?

Reply:
@divyarajmasani You can keep `Studio White` and `Cavernous Gray` only. I just revisited this template's design and noticed that every similar pair of colour is used on individual pages thus not really forming a palette of consistent colours used across this template. 

I've also updated the Figma design to reflect this change.

Reply:
@swissspidy 
There are these many colours because all pages uses different colours and there isn't exactly common palette for the template. However I've reduced some of the colours.

@maitreyie-chavan I've kept 4 colours shown on the first screen, which are also used on other pages.

![Screenshot 2021-07-26 at 3 18 16 PM](https://user-images.githubusercontent.com/6906779/126971044-837cf131-2282-4373-80b8-05e8d5d36ff4.png)

Reply:
I think that's much cleaner, thank you 👍